src/playlist-loader.js

```diff
@@ -338,76 +340,90 @@ const PlaylistLoader = function(srcUrl, withCredentials) {
     });
   });
 
-  // request the specified URL
-  request = XhrModule({
-    uri: srcUrl,
-    withCredentials
-  }, function(error, req) {
-    let parser;
-    let playlist;
-    let i;
+  loader.pause = () => {
+    loader.stopRequest();
+    window.clearTimeout(mediaUpdateTimeout);
+  };
 
-    // clear the loader's request reference
-    request = null;
+  loader.resume = () => {
```

Comment:
Similar to my comment above, what do you think about calling this `load()` to align with segment loader?